Dockerfile

```diff
@@ -63,6 +63,18 @@ RUN npm install yarn -g
 
 RUN echo "gem: --no-ri --no-rdoc --no-document" > /root/.gemrc
 
+COPY Add-js-rb-filtering-on-top-of-4.19.1.patch .
+
+RUN curl -o brp-strip https://raw.githubusercontent.com/rpm-software-management/rpm/rpm-4.19.1-release/scripts/brp-strip
+
+RUN chmod +x brp-strip
+
+RUN patch -p2 < Add-js-rb-filtering-on-top-of-4.19.1.patch && \
+  ls -l /usr/lib/rpm/brp-strip && \
+  mv brp-strip /usr/lib/rpm/brp-strip && \
+  ls -l /usr/lib/rpm/brp-strip && \
+  rm -f Add-js-rb-filtering-on-top-of-4.19.1.patch
```

Comment:
Nice!  A few comments here:

- The patch file should probably live in the container-assets directory, as opposed to the root.  That way you can reuse the `COPY . /build_scripts` command which already copies the container-assets directory.
- Prefer curling the file directly into /usr/lib/rpm/brp-strip, overwriting it, and then patching /usr/lib/rpm/brp-strip directly.
- No need to remove the patch file (it doesn't really get deleted from the container anyway)